editor/src/components/canvas/remix/utopia-remix-root-component.tsx

```diff
@@ -75,19 +75,25 @@ function useGetRouteModules(basePath: ElementPath) {
         continue
       }
 
+      const createExecutionScope = () =>
```

Comment:
Does this need to be a function that then gets invoked twice?

Reply:
Unfortunately it does. It's not quite being invoked twice, it's only being invoked when evaluating either the `default` or the `ErrorBoundary`. The reason is so that we can use the current project contents etc at the time of evaluating either of those